modules/microshift-gathering-sos-report.adoc

```diff
@@ -0,0 +1,47 @@
+// Module included in the following assemblies:
+
+// * microshift_support/microshift-sos-report
+
+:_content-type: PROCEDURE
+[id="gathering-data-microshift-sos-report_{context}"]
+= Gathering data from a {product-title} sos report
+
+.Prerequisites
+
+* You must have the `sos` package installed.
+
+.Procedure
+
+. Log into the failing host with your credintials.
+
+. To view the available profiles, run the following command:
++
+[source,terminal]
+----
+$ sos report --list-profiles
+----
+//not sure if the above step is needed. Need verification.
+. Perform the debug report creation procedure by running the following command:
++
+[source,terminal]
+----
+$ sos report --batch --clean --all-logs --profile microshift
+----
++
+.Example output
+[source,terminal]
+----
+sosreport (version 4.3)
```

Comment:
We should show output from 4.5.1 or later, since that's what we require. @pacevedom can you help with that?

Reply:
Thank you!